example/App_Intip.res

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable import/no-webpack-loader-syntax */
```

Comment:
@adiatma can you try this and give me some feedback ? 

if you want to try, clone this or fork and checkout `dev`, then `yarn add path_to_clone_dir` in your project.

Reply:
Hi, bro @ri7nz thanks for mentioning me in here,

After [fork](https://github.com/adiatma/rescript-mapbox-gl) this repo, and clone to my computer, then trying to `yarn add <path/rescript-mapbox-gl>`, and add dependencies to `bsconfig.json` at `intip-chrome-extensions`. but I found this error after implementing it to replace  [https://github.com/adiatma/intip-chrome-extensions/blob/master/src/Map.res](https://github.com/adiatma/intip-chrome-extensions/blob/master/src/Map.res).

```bash
>>>> Start compiling
bsb: [2/6] src/Map.d
FAILED: src/Map.d
FAILED: Map has a self cycle
FAILED: cannot make progress due to previous errors.
>>>> Finish compiling(exit: 1)
```

thanks bro @ri7nz

Reply:
Cause in `Mapbox` have module named `Map` and your implementation is `Map` that's make dependency cycle cause `Map` call it self.

![image](https://user-images.githubusercontent.com/16365952/112796456-31ccd880-9094-11eb-8f9d-f98d10734b7a.png)
![image](https://user-images.githubusercontent.com/16365952/112796616-6d67a280-9094-11eb-9046-e335701a5dfb.png)

It's work when your change `Map.res` to another name (e.g: `MapIntip.res`) and called `<MapIntip .../>`

Thank you @adiatma

Reply:
I see, thanks bro